server/dataset/dataset_metadata.py

```diff
@@ -34,11 +34,7 @@ def infer_dataset_s3_uri(server_config: ServerConfig, dataset_root: str, dataset
     Use the dataset_root, dataset_id, and the server config to infer the physical S3 URI for an Explorer dataset
     artifact
     """
-    # TODO @mdunitz remove after fork, update config to remove single_dataset option, the multiroot lookup will need to
-    #  remain while we support covid 19 cell atlas.
     #   See ticket https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/corpora-data-portal/1281 # noqa
```

Comment:
Not sure that this issue reference is useful to understanding. Remove? If should keep, at least update to the correct ticket (it moved repos):
```
    #   See ticket https://github.com/chanzuckerberg/single-cell-explorer/issues/119 # noqa
```